src/main/java/com/teamzunior/stringcalculator/Interpreter.java

```diff
@@ -0,0 +1,85 @@
+package com.teamzunior.stringcalculator;
+
+import org.springframework.util.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.teamzunior.stringcalculator.Operator.*;
+
+public class Interpreter {
+    private static final Pattern pattern = Pattern.compile("(.+)\\b([+\\-*/])(-?\\d+)");
+    private static final Map<String, Operator> operators;
+
+    private static final int FORMULA_INDEX = 1;
+    private static final int OPERATOR_INDEX = 2;
+    private static final int OPERAND_INDEX = 3;
+
+    public static Formula interpret(String formula) {
+        Matcher matcher = validateFormulaAndExtractMatcher(formula);
```

Comment:
각각의 함수로 나누어서 실행해도 될 것 같은데 하나의 함수로 `validate`을 `extractMatcher`와 함께 하나의 함수로 만드신 이유가 따로 있는 걸까요?
```suggestion
		validateFormula(formula);
        Matcher matcher = extractMatcher(formula);
```

Reply:
위에서도 바울님이 질문주셔서 남겼지만.. 처음에는 pattern.match(formula) 가 일종의 유효성 검증도 한다고 생각했고 그래서 같이 묶었었는데, 보니까 그게 아니더라구요. 여러분들 말씀처럼 따로 분리하는게 좋아보여요